cpp/tests/copying/scatter_to_tables_tests.cu

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
Try renaming this file to .cpp if it doesn't contain any __device__ code, to save compile time.

Reply:
Hm, good point. I didn't really think about it since the file I ported from was .cu, but you're right I don't think there's any device code to be compiled here.

Reply:
Yeah this is what I was saying in the standup on  Thursday. We used to have to make tests in .cu files because the old `column_wrapper` depended on `thrust::device_vector.h` which contains `__device__` code. But the new column wrappers intentionally avoid device code so our API tests don't need to be .cu, to reduce compile time.

Reply:
Done, works fine